iosconfsg2019/Schedule Tab/DetailGraphqlViewController.swift

```diff
@@ -406,9 +407,18 @@ class DetailGraphqlViewController: BaseViewController {
     }
 
     private func setupFeedbackButton() {
-        if let firstSpeaker = talk?.speakers.first, firstSpeaker.name != "Organiser" {
-            self.navigationItem.rightBarButtonItem = UIBarButtonItem(title: "Feedback", style: .plain, target: self, action: #selector(giveFeedback(_:)))
+        Flagsmith.shared.getValueForFeature(withID: "submit_feedback") { [weak self] result in
+            guard let self else { return }
+            switch result {
+            case let .success(.bool(canSubmitFeedback)):
+                if canSubmitFeedback, let firstSpeaker = self.talk?.speakers.first, firstSpeaker.name != "Organiser" {
```

Comment:
if `canSubmitFeedback` = false, don't add the Submit Feedback button.

Post conference, we don't need to show this button.